packages/components/src/toggle-group-control/toggle-group-control-option-base/styles.ts

```diff
@@ -64,6 +64,30 @@ export const buttonView = ( {
 		border: 0;
 	}
 
+	&[disabled],
+	&[aria-disabled='true'] {
+		cursor: default;
+		${ ButtonContentView } {
+			opacity: 0.4;
+		}
```

Comment:
Opacity is applied to the button contents, so that the focus ring can be at full opacity